wiki/en/en-Software-Manual.md

```diff
@@ -4,104 +4,94 @@ title: "Software Manual"
 lang: "en"
 permalink: "/wiki/Software-Manual"
 ---
-# Jamulus Software Manual
+# Jamulus User Manual
 
-## Main Window
+{:.no_toc}
```

Comment:
What's this class?

Reply:
No idea what that is - it's in the EN doc.

Reply:
Oh and yes, it builds and displays correctly building it locally.